### PR TITLE
Fixed #34109 -- How to use Uvicorn without Gunicorn

### PR DESCRIPTION
**Pull request by simonw**
https://code.djangoproject.com/ticket/34109#ticket

### CODE REVIEW

**Comment by simonw** [comment]
Lots of relevant discussion going on here: https://code.djangoproject.com/ticket/34109

Short version: the `gunicorn` bits were added deliberately here:

- https://github.com/django/django/pull/13455

To reflect recommendations in the Uvicorn documentation here: https://www.uvicorn.org/deployment/#gunicorn

**Comment by simonw** [comment]
The title of the page at https://docs.djangoproject.com/en/4.1/howto/deployment/asgi/uvicorn/ is "How to use Django with Uvicorn".

As such, I think it's worth explaining both the with-gunicorn and without-gunicorn mechanisms. I'll update my PR to do that instead.

File: docs/howto/deployment/asgi/uvicorn.txt [hunk 1]
```diff
@@ -24,12 +24,30 @@ called (separated by a colon).
 
 For a typical Django project, invoking Uvicorn would look like::
 
-    gunicorn myproject.asgi:application -k uvicorn.workers.UvicornWorker
+    uvicorn myproject.asgi:application
```
Comment (pauloxnet) [inline docs/howto/deployment/asgi/uvicorn.txt:27]: We can also suggest here to use `uvicorn` as a module?
```suggestion
    python -m uvicorn myproject.asgi:application
```

File: docs/howto/deployment/asgi/uvicorn.txt [hunk 2]
```diff
@@ -24,12 +24,30 @@ called (separated by a colon).
...
+Gunicorn_ is a robust web server that implements process monitoring and automatic
+restarts. This can be useful when running Uvicorn in a production environment.
+
+To install Uvicorn and Gunicorn, use the following::
+
+    python -m pip install uvicorn gunicorn
+
+Then start Gunicorn using the Uvicorn worker class like this::
+
+    gunicorn myproject.asgi:application -k uvicorn.workers.UvicornWorker
```
Comment (pauloxnet) [inline docs/howto/deployment/asgi/uvicorn.txt:50]: We can also suggest here to use `gunicorn` as a module?
```suggestion
    python -m gunicorn myproject.asgi:application -k uvicorn.workers.UvicornWorker
```

File: docs/howto/deployment/asgi/uvicorn.txt (hunk 1 above)
Comment (simonw) [reply docs/howto/deployment/asgi/uvicorn.txt:27]: Yup, this works! Thanks.

**Review by carltongibson — APPROVED** [review]
OK, thanks @simonw — let's go with this, as handling both the gunicorn and container-like options. 

(🤷 :)

**Pull request merged**